code/modules/organs/robolimbs.dm

```diff
@@ -172,6 +172,22 @@ var/datum/robolimb/basic_robolimb
 	species_cannot_use = list()
 	applies_to_part = list(BP_L_ARM, BP_R_ARM, BP_L_LEG, BP_R_LEG, BP_L_FOOT, BP_R_FOOT, BP_L_HAND, BP_R_HAND)
 
+
+/datum/robolimb/cenilimisybernetics
+	company = "Cenilimi Cybernetics"
+	desc = "This prosthetic is created by a Teshari-owned company, for Teshari."
+	icon = 'icons/mob/human_races/cyberlimbs/cenilimicybernetics/cenilimicybernetics.dmi'
```

Comment:
icon = 'icons/uristmob/species, make a resomi/tesh folder, add relevant icon = changes to support it.